db/sample_data/events.rb

```diff
@@ -74,7 +74,7 @@ def event_gongakrobatik
       end_date: Date.new(2021, 03, 05)
   )
   Event.new(
-      name: 'Einführung in die Kunst der Gongakrobatik',
+      name: 'Gongakrobatik',
```

Comment:
So it is more or less just a presentation thing? Future users might still choose a long name...